app/assets/javascripts/angular/resources/shared_graph_behavior.js

```diff
@@ -21,14 +21,15 @@ angular.module("Prometheus.services").factory("SharedGraphBehavior", ["$http", "
       $scope.widgets = $scope.widgets || [];
       var urlVars = URLVariablesDecoder();
       var templateVarRe = /^var\.(.*)$/;
-      for (var o in urlVars) {
-        var matches = o.match(templateVarRe)
+      var o;
```

Comment:
javascript scoping rules :/ the o is actually available outside of the `for-in` loop, even in the original declaration `for (var o in urlVars)`